public/views/applicationForm.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>MindSet</title>
+    <link rel="stylesheet" href="../styles/reset.css">
+    <link rel="stylesheet" href="../styles/styles.css">
+    <link rel="stylesheet" href="../styles/modal.css">
+    <link rel="stylesheet" href="../styles/application.css">
+    <script src="../js/modal.js" async></script>
+    <script src="../js/applicationForm.js" async></script>
+  </head>
+  <body>
+    <header>
+      <a id="mindSetLogo" href="/"><span>Mind</span>SET</a>
+      <nav>
+        <a id="adminsNav" href="/views/adminList.html">admins</a>
+        <a id="applicationsNav" href="/views/applicationList.html">applications</a>
+        <a id="clientsNav" href="/views/clientList.html">clients</a>
+        <a id="interviewsNav" href="/views/interviewList.html">interviews</a>
+        <a id="positionsNav" href="/views/positionList.html">positions</a>
+        <a id="profilesNav" href="/views/profileList.html">profiles</a>
+        <a id="psychologistsNav" href="/views/psychologistList.html">psychologists</a>
+        <a id="sessionsNav" href="/views/sessionList.html">sessions</a>
+      </nav>
+    </header>
+    <div class="container">
+      <h2 class="header">Applications Form</h2>
+      <form id="form">
+        <label for="position">Position:</label>
+        <select id="position" name="position" required></select>
+        <label for="postulant">Postulant:</label>
+        <select id="postulant" name="postulant" required></select>
+        <label for="interview">Interview:</label>
+        <select id="interview" name="interview"></select>
```

Comment:
see how a date is handled [here](https://github.com/BaSD-2021/Esteban-MindSet-server/pull/45/files#diff-efdbd9ee8c4229255aa369e839207b2f380fec7b405af942426dd30a7bb255f1R23)